exercises/nodejs/src/game.js

```diff
@@ -1,172 +1,198 @@
-import generator from 'random-seed'
+import generator from "random-seed";
 
-var Game = function () {
-  var players = new Array()
-  var places = new Array(6)
-  var purses = new Array(6)
-  var inPenaltyBox = new Array(6)
+let Game = function () {
+  const players = new Array();
+  const boardPositions = new Array(6);
+  const purses = new Array(6);
+  const inPenaltyBox = new Array(6);
```

Comment:
Do you think number 6 could be a magic number? You could maybe store it in a variable

Reply:
Change made